recipe/meta.yaml

```diff
@@ -1,45 +1,44 @@
 {% set name = "dm-tree" %}
-{% set version = "0.1.5" %}
+{% set version = "0.1.7" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dm-tree-{{ version }}.tar.gz
-  sha256: a951d2239111dfcc468071bc8ff792c7b1e3192cab5a3c94d33a8b2bda3127fa
-  patches:
-    - patches/0001-Contain-bazel-root.patch
-    - patches/0002-cxxopt-for-osx-sysroot.patch   # [osx]
+  sha256: 30fec8aca5b92823c0e796a2f33b875b4dccd470b57e91e6c542405c5f77fd2a
 
 build:
   number: 0
   skip: true   #[ppc64le]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 
 # Need these up here for conda-smithy to handle them properly.
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - python
-    - bazel
-    - m2-patch  # [win]
-    - patch     # [not win]
+    - cmake
+    - make  # [not win]
+    - git
```

Comment:
I think we are supposed to not use `git` on Windows, but use `m2-git` instead. It could result in issues with Prefect.